schemas/context/store.schema.json

```diff
@@ -0,0 +1,132 @@
+        },
+        "xdm:description": {
+          "title": "Description",
+          "type": "string",
+          "description": "The description of the store."
+        },
+        "xdm:parentStoreId": {
+          "title": "Parent Store Id",
+          "type": "string",
+          "description":
+            "Id of the store this store is contained or closely related to, e.g a pharmacy inside a supermarket store."
+        },
+        "xdm:brand": {
+          "title": "Brand",
+          "type": "string",
+          "description": "Store brand or label."
+        },
+        "xdm:location": {
+          "title": "Location",
+          "$ref": "https://ns.adobe.com/xdm/context/place",
+          "description": "The perminant physical location of the store."
+        },
+        "xdm:generalEmail": {
+          "title": "General Email",
+          "$ref": "https://ns.adobe.com/xdm/context/emailaddress",
+          "description": "The general email address for the store."
+        },
+        "xdm:openingHours": {
+          "title": "Opening Hours",
+          "$ref": "https://ns.adobe.com/xdm/context/openinghours",
+          "description": "Store opening hours."
+        },
+        "xdm:manager": {
+          "title": "Person",
+          "$ref": "https://ns.adobe.com/xdm/context/person",
+          "description": "Manager of the store."
+        },
+        "xdm:dmaId": {
+          "title": "Designated Market Area",
+          "type": "integer",
+          "description":
+            "The Nielsen Media Research designated market area the store serves."
+        },
+        "xdm:market": {
+          "title": "Market",
+          "type": "string",
+          "description": "Sales organization  market area the store serves."
+        },
+        "xdm:openingDate": {
+          "title": "Original Opening Date",
+          "type": "string",
+          "format": "date",
+          "description":
+            "The date that the store first opened for business, even if the address changed, the area served remained the same."
+        },
+        "xdm:remodelDate": {
+          "title": "Last Remodel Date",
+          "type": "string",
+          "format": "date",
+          "description":
+            "The date that the store last had a significant remodel."
+        },
+        "xdm:revenueBand": {
+          "title": "Revenue Band",
+          "type": "string",
+          "description": "Revenue classification for this store."
+        },
+        "xdm:model": {
+          "title": "Store Model",
+          "type": "string",
+          "description":
+            "The model or type of store. Examples: online, express, micro, kiosk, partner, superstore, mobile, distribution center."
+        },
+        "xdm:features": {
+          "title": "Store Features",
+          "type": "array",
+          "description":
+            "List of non-standard features that the store has, such as product demo areas, e.g Apple displays, Bike Shop etc.",
```

Comment:
Again, we need a `meta:enum` of known features. We can probably get examples of feature lists from OpenStreetMap, Gogle Maps, Facebook Places, Foursquare, Yelp and merge and consolidate them.